benchmarks/bench.sh

```diff
@@ -247,6 +257,22 @@ run_tpch_mem() {
     $CARGO_COMMAND --bin tpch -- benchmark datafusion --iterations 5 --path "${DATA_DIR}" -m --format parquet -o ${RESULTS_FILE}
 }
 
+# Runs the parquet filter benchmark
+run_parquet() {
+    RESULTS_FILE="${RESULTS_DIR}/parquet.json"
+    echo "RESULTS_FILE: ${RESULTS_FILE}"
+    echo "Running parquet filter benchmark..."
+    $CARGO_COMMAND --bin parquet -- filter --path "${DATA_DIR}" --scale-factor 1.0 --iterations 5 -o ${RESULTS_FILE}
```

Comment:
why we dont apply scale-factor for tpch? is it 1 by default?

Reply:
This is even more confusing -- the scale factor for tpch is actually a applied, but it is applied when we create the data. Specifically

https://github.com/apache/arrow-datafusion/blob/2787e7a36a6be83d91201df20827d3695f933300/benchmarks/bench.sh#L202

The "parquet" benchmark actually doesn't use the tpch dataset at all, and instead generates its own data. Hwever, it overloads the "scale factor" terminology to describe the relative sizes